### DISCUSSION TITLE
Late submission question

### DISCUSSION THREAD

**Discussion by y1bo44** [cooperrc/computational-mechanics#47] [General]
I am a little bit overwhelmed by the homwork,just wanna ask will we okay to submit a little bit late?like 1 day.The syllabus says there are no late penalty.

**Comment by feliciones** [comment]
Yeah, just send him an email letting him know. Also, upload and submit what you have to google classroom so he can put comments on it and help you. You can always resubmit for a regrade.

**Reply by y1bo44** [reply]
thanks
